### PR TITLE
Added refresh tokens

### PR DESCRIPTION
**Pull request by alberto-kodeacross**

### CODE REVIEW

File: src/auth/providers/useSignInWithTheKey.ts [hunk 1]
```diff
@@ -105,9 +106,11 @@ export const useSignInWithTheKey = () => {
 
   const makeAccessTokenRequest = useCallback(
     async (request: ApiRouteConfigEntry, params: string) => {
+      console.log('request', request);
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:109]: Please delete logger.

File: src/auth/providers/useSignInWithTheKey.ts [hunk 2]
```diff
@@ -105,9 +106,11 @@ export const useSignInWithTheKey = () => {
...
       const { access_token, refresh_token } = ((await dispatch(
         callApi(request, {}, params),
       )) as unknown) as { access_token: string; refresh_token?: string };
+      console.log(`refresh_token from the key`, refresh_token);
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:113]: Please delete logger.

**Review by SeijiV13 — DISMISSED** [review]

File: src/auth/providers/useSignInWithTheKey.ts [hunk 3]
```diff
@@ -181,18 +184,26 @@ export const useSignInWithTheKey = () => {
 
       try {
         const accessToken = await getAccessToken(options);
+        console.log('accessToken', accessToken);
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:187]: Please delete logger.

File: src/auth/providers/useSignInWithTheKey.ts [hunk 4]
```diff
@@ -181,18 +184,26 @@ export const useSignInWithTheKey = () => {
...
         const ticket = await getTicket(accessToken);
+        console.log('ticket', ticket);
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:189]: Please delete logger.

File: src/auth/providers/useSignInWithTheKey.ts [hunk 5]
```diff
@@ -181,18 +184,26 @@ export const useSignInWithTheKey = () => {
...
 
         const anonymousUid = await getAnonymousUid();
+        console.log('anonymousUid', anonymousUid);
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:192]: Please delete logger.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 1 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 2 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 3 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 4 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 5 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts [hunk 6]
```diff
@@ -190,9 +191,13 @@ export const useSignInWithTheKey = () => {
             anonymousUid,
           },
         });
+        // @ts-ignore
```
Comment (rossellemabunga) [inline src/auth/providers/useSignInWithTheKey.ts:194]: Remove if not needed.

File: src/auth/providers/useSignInWithTheKey.ts [hunk 7]
```diff
@@ -181,18 +184,26 @@ export const useSignInWithTheKey = () => {
...
         const anonymousUid = await getAnonymousUid();
+        console.log('anonymousUid', anonymousUid);
         const { data } = await apiSignInWithTheKey({
           variables: {
             accessToken: ticket,
             anonymousUid,
           },
         });
+        // @ts-ignore
+        console.log(`refresh_token from graphql`, data);
```
[the remark this answers is not in the document]
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Logger removed.

File: src/auth/providers/useSignInWithTheKey.ts (hunk 6 above)
Comment (uriel54321) [reply src/auth/providers/useSignInWithTheKey.ts]: Done removing the comment

**Review by rossellemabunga — APPROVED** [review]

**Review by SeijiV13 — APPROVED** [review]

**Review by rossellemabunga — APPROVED** [review]

**Pull request merged**